web-previewer/src/components/useRouteParameterSync.js

```diff
@@ -0,0 +1,17 @@
+import { watch, onMounted } from "vue";
+import { useRoute, useRouter } from "vue-router";
+
+export default function useRouteParameterSync(previewState) {
+  const route = useRoute();
+  const router = useRouter();
+
+  onMounted(() => {
+    if (route.query.id) {
+      previewState.developerId.value = route.query.id;
```

Comment:
Sorry if my comment in previous review was incorrect. 

I actually meant for the route param to be `developerId` - not change the name of the variable in `usePreviewState`.

On second thought maybe it should just be `developer`, given that's what it's called in code + interface. My bad!

Reply:
Ohh whoops! Alright I'll be making those changes.

And no worries!